### DISCUSSION TITLE
11월 18일 (목)

### DISCUSSION THREAD

**Discussion by InChil2** [InChil2/Detection_of_lesions#8] [회의록]
회의 시작

**Comment by Leemc95** [comment]
모델이 예측한 박스를 시각화하여 확인하기

**Comment by pkwon35** [comment]
정밀도 vs 재현율   어떤 상황에 뭐가 높아야할까?

정밀도와 재현율 중 어느 것이 더 중요한지는 경우에 따라 다르다. 

1) **암 판단 모델**과 같이 실제 Positive(양성)인 데이터를 Negative(음성)로 잘못 판단하는 것이 큰 문제인 경우 재현율이 상대적으로 더 중요해지며

2) **스팸메일 분류모델**과 같이 실제 Negative(일반메일)인 데이터를 Positive(스팸메일)로 잘못 판단하는 것이 큰 문제인 경우 정밀도가 상대적으로 더 중요해지는 것이다. 물론 가장 좋은 것은 두 수치 모두 높은 경우일 것이며, 한 수치가 상대적으로 낮더라도 그 차이가 심해서는 안된다.

**Comment by Choihankyul** [comment]
1. 양질의 데이터를 선별해서 학습을 해야하는가?
2. 라벨(병명)에 대해 공부 필요
3. 참고 https://herbwood.tistory.com/2

**Comment by Leemc95** [comment]
<YOLOv5 학습 블로그>

https://lynnshin.tistory.com/47

**Comment by Leemc95** [comment]
<EfficientNet>

https://github.com/google/automl/tree/master/efficientdet
